lib/iris/tests/unit/fileformats/nc_load_rules/actions/test__grid_mappings.py

```diff
@@ -830,5 +830,32 @@ def test_aux_lat_rotated(self):
         self.check_result(result, xco_is_aux=True, xco_no_cs=True)
 
 
+class Test__nondimcoords(Mixin__grid_mapping, tests.IrisTest):
```

Comment:
Note: this code was moved here unchanged from the former file 
`lib/iris/tests/unit/fileformats/nc_load_rules/actions/test__additional.py`, now removed